src/utils/getRandomNumber.js

```diff
@@ -0,0 +1,5 @@
+import { Random } from '@woowacourse/mission-utils';
+
+export const getRandomNumber = () => {
+  return Random.pickNumberInRange(0, 9);
+};
```

Comment:
```suggestion
export const getRandomNumber = (min, max) => {
  return Random.pickNumberInRange(min, max);
};
```
랜덤 숫자의 범위를 인자로 받으면 추후에 요구사항이 변경되어도 유연하게 동작할 수 있을 것 같아요~!